README.md

```diff
@@ -50,27 +67,30 @@ yay -S nishanths-license-git
 
 To print a license to stdout, run the `license` command followed by the license name:
 
-```
+```powershell
```

Comment:
Please remove the powershell syntax for the code blocks, or replace them with just `sh`.